tests/queries/0_stateless/03165_database_and_table_count_limitation_test.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+
+#Tags: shared, no-parallel
+
+# Get the current script directory
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+BASE="$CUR_DIR/$(basename "${BASH_SOURCE[0]}" .sh)"
+
+# Load shell_config.sh
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+# remove --database=$CLICKHOUSE_DATABASE from $CLICKHOUSE_CLIENT
+CLICKHOUSE_CLIENT=$(echo $CLICKHOUSE_CLIENT | sed "s/--database=$CLICKHOUSE_DATABASE//")
+
+# Create a temporary directory
+TEMP_DIR=$(mktemp -d "$CUR_DIR/$(basename "$BASE").XXXXXX")
+
+# Generate config.xml
+CONFIG_FILE="$TEMP_DIR/config.xml"
+cat > "$CONFIG_FILE" <<EOL
+<yandex>
```

Comment:
1. We have left yandex :)
2. Don't do this in stateless test :(  we write integration tests to test special config